### DISCUSSION TITLE
getting Uncaught JsonException for Webhook

### DISCUSSION THREAD

**Discussion by fossildj** [nutgram/nutgram#334] [Q&A]
i set running mode to webhook, but when i open my php file in browser, i get the following error:

Fatal error: Uncaught JsonException: Syntax error in C:\xampp\htdocs\bots\testbot\vendor\nutgram\nutgram\src\RunningMode\Webhook.php:62 Stack trace: #0 C:\xampp\htdocs\bots\testbot\vendor\nutgram\nutgram\src\RunningMode\Webhook.php(62): json_decode('', NULL, 512, 4194304) #1 C:\xampp\htdocs\bots\testbot\vendor\nutgram\nutgram\src\Nutgram.php(228): SergiX44\Nutgram\RunningMode\Webhook->processUpdates(Object(SergiX44\Nutgram\Nutgram)) #2 C:\xampp\htdocs\bots\testbot\test.php(43): SergiX44\Nutgram\Nutgram->run() #3 {main} thrown in C:\xampp\htdocs\bots\testbot\vendor\nutgram\nutgram\src\RunningMode\Webhook.php on line 62

```
use SergiX44\Nutgram\Nutgram;
use SergiX44\Nutgram\RunningMode\Webhook;

$bot = new Nutgram('xxxxxxxxx:AAEzCTG0UlpKnLGvOddKiodjFSwmJmk4Pnw');
$bot->setRunningMode(Webhook::class);
```

**Comment by Lukasss93** [comment]
You don't have to be the one to visit your php file. You have to register the webhook that Telegram will use to call your bot.
https://core.telegram.org/bots/api#setwebhook

**Reply by fossildj** [reply]
> You don't have to be the one to visit your php file. You have to register the webhook that Telegram will use to call your bot. https://core.telegram.org/bots/api#setwebhook

i have already set webhook via the api.telegram url and my bot is working fine with nutgram. just wondering why when i visit my php file, it has those errors, i usually visit my php file to make sure theres no error with the coding. if theres nothing wrong, it will show a blank page.
